### PR TITLE
Clarify re-key readme and split EET helper into two suites

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Summary of the change**:
Add warnings to the re-key README and avoid a EET helper failure mode by splitting into two suites.

### CODE REVIEW

File: test-clients/src/main/java/com/hedera/services/bdd/suites/misc/FreezeRekeyedState.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+import java.util.List;
+import java.util.Map;
+
+import static com.hedera.services.bdd.spec.HapiApiSpec.customHapiSpec;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.freeze;
+import static com.hedera.services.bdd.suites.misc.RekeySavedStateTreasury.newTreasuryStartUpAccountLoc;
+
+/**
+ * Given a network with a "re-keyed" treasury account, we want to use this the
```
Comment (ljianghedera) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/misc/FreezeRekeyedState.java:36]: Remove either `this` or `the`.

**Review by ljianghedera — APPROVED** [review]
LGTM except the small comment.

**Review by anighanta — APPROVED** [review]
LGTM, but what made you move the freeze operation to a separate suite ?

**Comment by tinker-michaelj** [comment]
> LGTM, but what made you move the freeze operation to a separate suite ?

I noticed a problem with the second spec in the suite failing to initialize in some edge case, so I just moved it to its own suite. 😁

**Pull request merged**